src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBtfBlocking.java

```diff
@@ -152,12 +152,12 @@ public void adsAtfDisableBtfExceptHighlyViewableSlotsOasis(String wikiName, Stri
     AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testedPage, DESKTOP_PAGE_SIZE);
     adsBaseObject.waitForPageLoadedWithGpt();
 
-    Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.MEDREC));
-    Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.TOP_LB));
-    Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.INVISIBLE_SKIN));
-    Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.INVISIBLE_HIGH_IMPACT_2));
-    Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.INCONTENT_LEADERBOARD));
-    Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.FLOATING_MEDREC));
+    Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.MEDREC), "TOP_RIGHT_BOXAD not loaded");
```

Comment:
• use `AdsContent.MEDREC` in assert message
e.g. `AdsContent.MEDREC` + “ not loaded”
• what about more descriptive message? 
e.g. “Ad is not loaded inside ” + `AdsContent.MEDREC`